### DISCUSSION TITLE
Classic and Classic Black Theme 3.5inch

### DISCUSSION THREAD

**Discussion by cactuz9** [mathoudebine/turing-smart-screen-python#609] [Themes]
I made this... Not much to say more.
These use monospace font called MorePerfectDOSVGA.ttf made by this guy https://laemeur.sdf.org/fonts/
Font is inside the package, make sure to place it in font folder.

![previe2](https://github.com/user-attachments/assets/9962b332-cd9a-45e7-8258-4d4f4d6dbfb4)

![preview](https://github.com/user-attachments/assets/7ea818ba-1915-4688-98ba-8d16bdce8f6a)

![06d5bf1](https://github.com/user-attachments/assets/26b9e0da-613b-46b4-a134-bf8a518603b5)

Place font here:
![font location](https://github.com/user-attachments/assets/a495da79-6922-4b1d-a1e5-1620244f2cbe)

Download:
[Classic3-5.zip](https://github.com/user-attachments/files/17836276/Classic3-5.zip)

[Classic3-5 1.1.zip](https://github.com/user-attachments/files/21018593/Classic3-5.1.1.zip)
1.1 - GPU memory text changed to GPU frequency.

Enjoy!

![06d5bf1](https://github.com/user-attachments/assets/26b9e0da-613b-46b4-a134-bf8a518603b5)

**Comment by luanmyke** [comment]
I made a landscape version of it, only for the black one, but if you guys want the colored version just copy the color codes from the classic one, and paste it.

![Screenshot 2024-12-28 003808](https://github.com/user-attachments/assets/0fdac191-ec21-4b05-af0a-9289102cc152)

[theme.zip](https://github.com/user-attachments/files/18264865/theme.zip)

**Reply by fearlesschicken** [reply]
hi, thanks for making the landscape version.  Having some trouble with the GPU text on mine.  Any text after U seems to be cut off.
<img width="640" height="480" alt="IMG_3181" src="https://github.com/user-attachments/assets/7a86ce1e-37b6-4e84-a331-955089775d3a" />
I'm using GPU frequency here but I've also test your version with GPU memory and it is the same result.

**Reply by louienux** [reply]
Hey, flip the background png to landscape. It's the background image covering the text.

**Reply by Miguel-JMZ** [reply]
hi, i have the same problem i tried but i cant  fix it, can upload a fixed version?

**Comment by sysyphus** [comment]
![photo_2025-02-01_20-52-27](https://github.com/user-attachments/assets/cbc984f9-f56d-4a80-a1ac-e45db73cb33f)

It's gorgeous!!

**Comment by Argenlune** [comment]
Could you change the GPU memory usage to GPU core usage for me?
The GPU memory usage isn't useful to me.

**Reply by cactuz9** [reply]
Sure, check main post.

**Reply by Argenlune** [reply]
![image](https://github.com/user-attachments/assets/11cab6de-8890-4ac3-a4f6-5da8d400c27e)

Thank you for your kind and your programming

**Reply by Argenlune** [reply]
Can I ask you something. I Ithink CPU graph is maybe frequency. What GPU graph is mean what is that? temperature?

**Reply by cactuz9** [reply]
Both graphs shows "PERCENTAGE" which is most probably CPU/GPU load. 
Values ​​between 0 and 100.

**Reply by Argenlune** [reply]
![20250711_003216](https://github.com/user-attachments/assets/ecc94667-7f46-4ddc-ad1c-e4f0ba20fa16)
When uploads and downloads exceed 1GB, they're shown as 'gm'. I contacted support because I thought something might have gone wrong.
